core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -35,7 +35,6 @@ import scala.jdk.CollectionConverters._
 import scala.collection._
 import scala.collection.mutable.ArrayBuffer
 import scala.util.{Failure, Success, Try}
-import org.apache.kafka.common.requests.{AbstractControlRequest, LeaderAndIsrRequest}
```

Comment:
It seems we can remove `AbstractControlRequest` in this PR.

Reply:
Good point - this can go if we move `Type` to `LeaderAndIsrRequest`.